ClientApp/src/app/charts/charts.component.html

```diff
@@ -0,0 +1,12 @@
+<dp-header4 class="dp-ml-40">Header 4</dp-header4>
+<div class="charts">
+  <dp-pie-chart
+    [series]="series"
+    [labels]="labels"
+    [showTooltip]="true"
+    [showLabels]="false"
+    [showLegend]="true"
+    [colors]="colors"
+    [size]="400"
+  ></dp-pie-chart>
+</div>
```

Comment:
The chart component should probably be moved to 'src/app/components/charts/...'
🗺️